torch/testing/_internal/dynamo_test_failures.py

```diff
@@ -7750,6 +7754,13 @@
     "TestProfiler.test_profiler_disable_fwd_bwd_link",
     "TestProfiler.test_profiler_metadata",
     "TestSerializeCustomClass.test_custom_class",
+    "TestTensorExprPyBind.test_kernel_with_transpose",
+    "TestTensorExprPyBind.test_kernel_with_custom_lowering",
+    "TestTensorExprPyBind.test_kernel_with_expand",
+    "TestTensorExprPyBind.test_kernel_with_permute",
+    "TestTensorExprPyBind.test_kernel_with_scalar_inputs",
+    "TestTensorExprPyBind.test_kernel_with_t",
+    "TestTensorExprPyBind.test_kernel_with_tensor_inputs",
```

Comment:
dynamo'ing TensorExpr (NNC?) code, uh oh 😛

Reply:
Just curious, why skip and not fail - flaky?

Reply:
flaky: it failed locally for me, but passes in CI